perllib/mySociety/MaPit.pm

```diff
@@ -32,29 +31,36 @@ sub configure (;$) {
 }
 
 my $ua;
-sub _call ($$;$) {
-    my ($url, $params, $opts) = @_;
+
+# Calls MapIt, returns the decoded JSON.
+# I do it properly with HTTP status codes, then I just ignore them!
```

Comment:
I'm really not sure what this comment means here.

Reply:
Yeah, just copied that across, but could be clearer! It means MapIt is returning e.g. 400 for bad request and 404 for postcode not found, but this library code ignores that entirely.